src/components/Bars/index.tsx

```diff
@@ -141,6 +141,18 @@ const Bars: React.FC<BarsProps> = (props) => {
     const instance = echarts.getInstanceByDom(chartDOM as any);
     if (instance) {
       instance.setOption(option);
+      instance.on('click', function (params) {
+        const { data, seriesName } = params;
+        if (seriesName == 'Activity') {
+          let [year, month] = data.toString().split(',')[0].split('-');
+          if (month.length < 2) {
+            month = '0' + month;
+          }
+          window.open(
+            `${window.location.href}/issues?q=updated:${year}-${month}`
```

Comment:
q=updated:${year}-${month} sort:updated-asc will be better in my opinion:

<img width="1294" alt="image" src="https://user-images.githubusercontent.com/32434520/190038917-c794705e-3311-47d9-96a3-0b4c0deb76f4.png">

And, since `src/components/Bars/index.tsx` is used by both RepoActInflTrend and DeveloperActInfTrend, so I think it is better to move this event handler outside of the file and pass it as a parameter to Bars component. If so, we can concatenate url string directly by `currentRepo` provided by RepoActiInflTrendView rather by `window.location.href`, which is not reliable I think.

Reply:
I moved the function to the RepoActInflTrend view, added a onClick function to the component Bar. The function is optinal, so it is compatiable to other views.